### DISCUSSION TITLE
Why does the macOS desktop version need Intel ?

### DISCUSSION THREAD

**Discussion by tao-bioinfo** [GMOD/jbrowse-components#4008] [General] [answered]
I am using an apple silicon device (M1 chip, ARM based ), and it seems the macOS desktop version (`jbrowse-desktop-v2.7.1-mac.dmg`) cannot be installed unless rosetta is present. However I do not want to install rosetta since it would bring many processes that consume battery very quickly.

Is there any reason why the macOS desktop version needs Intel instead of universal binary?

**Comment by cmdcolin** [comment, marked as answer]
I think this was just an oversight on our part, I don't have a M1 mac myself to test so wasn't even aware of the issue. I will make an issue from this and we can try to make a native binary

**Comment by carolinebridge** [comment]
https://drive.google.com/file/d/1UdpCNlcXrhvv514NGVx16Ywgb0906bGn/view?usp=sharing

.dmg for jbrowse desktop 2.7.1

**Reply by cmdcolin** [reply]
@billzt we tried creating a universal macos build with the above drive link, would you be able to test it out?

**Reply by tao-bioinfo** [reply]
@cmdcolin Thank you. I have tried viewing reference sequences and read alignments (BAM). Perfect.

**Reply by cmdcolin** [reply]
@billzt v2.7.2 now released with the universal builds, thanks for reporting this!
